src/main/java/org/sagebionetworks/bridge/models/permissions/PermissionDetail.java

```diff
@@ -0,0 +1,76 @@
+package org.sagebionetworks.bridge.models.permissions;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.sagebionetworks.bridge.models.accounts.AccountRef;
+
+import java.util.Objects;
+
+@JsonPropertyOrder({ "guid", "userId", "accessLevel", "entityType", "entityId", "entityRef", "userAccountRef" })
+public final class PermissionDetail {
+    
+    private final String guid;
+    private final String userId;
```

Comment:
It seems like userId should be covered by AccountRef, which I would call "account" to be consistent with EnrollmentDetail where we use the property "participant".

Reply:
Makes sense, removed userId and renamed the account ref to "account".